README.md

```diff
@@ -79,73 +97,104 @@ Fields that can be included in the response body for `GET /`.
+| `GET /products/{productId}/order-parameters`             | Core                | -                                                                  | JSON Schema                                                                  |                                                                                                                                                                                                                                     |
+| `GET /orders`                                            | Core                | -                                                                  | [Orders Collection](./order/README.md#order-collection)                      |                                                                                                                                                                                                                                     |
+| `GET /orders/{orderId}`                                  | Core                | -                                                                  | [Order Object](./order/README.md#order-pobject)                              |                                                                                                                                                                                                                                     |
+| `POST /products/{productId}/orders`                      | Core                | [Order Request](./order/README.md#order-request) or any object     | -                                                                            | Order a capture with a particular set of [parameters](https://github.com/Element84/stapi-spec/blob/main/product/README.md#parameters) as defined in the products or a request that was provided through the opportunities endpoint. |
+| `POST /products/{productId}/opportunities`               | Opportunities       | [Opportunity Request](./opportunity/README.md#opportunity-request) | Sync search: [Opportunities Collection](./opportunity/README.md#opportunities-collection); Async search: [Opportunity Search Record](./opportunity/README.md#opportunity-search-record) | Explore the opportunities available for a particular set of [parameters](https://github.com/Element84/stapi-spec/blob/main/product/README.md#parameters) |
+| `GET /products/{productId}/opportunities/{collectionId}` | Async Opportunities | -                                                                  | [Opportunities Collection](./opportunity/README.md#opportunities-collection) | Get the opportunity collection for an async opportunity search              |
+| `GET /searches/opportunities/`                           | Async Opportunities | -                                                                  | [[Opportunity Search Record](./opportunity/README.md#opportunity-search-record)] |          |
```

Comment:
I'm unclear as to what this endpoint does. Why does it have a trailing slash? and why does it return the same type as /searches/opportunities/{searchRecordId}?

Reply:
It's hard to see in the raw form but this endpoint is returning an array of Opportunity Search Records. This endpoint allows a user to list all their searches in one place. The trailing slash was an error on my part and I will remove it.